spec/features/sign_in_spec.rb

```diff
@@ -0,0 +1,54 @@
+require "rails_helper"
+
+describe "sign in", :type => :feature do
+  scenario "user enters a blank email" do
+    user = FactoryGirl.create(:user)
+
+    visit '/login'
+
+    fill_in('email', :with => '')
+    fill_in('password', :with => user.password)
+    click_button('login')
+
+    expect(current_path).to eq('/login')
+    expect(page).to have_content 'Invalid email or password'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.